security-framework/src/identity.rs

```diff
@@ -51,3 +51,22 @@ impl SecIdentity {
         }
     }
 }
+
+unsafe impl Send for SecIdentity {}
+
+
+#[cfg(test)]
+mod test {
+    use std::ptr;
+
+    use super::SecIdentity;
+
+    #[test]
+    fn identity_has_send_bound() {
```

Comment:
If you cfg-off the test to OSX, you can get a `SecIdentity` here: https://github.com/sfackler/rust-security-framework/blob/master/security-framework/src/os/macos/mod.rs#L21, but an easier way is to do something like this:

```rust
fn assert_send<T: Send>() {}

assert_send::<SecIdentity>();
```